tests/ops/functions/test_eigvals.py

```diff
@@ -106,7 +106,7 @@ def test_ctrl(self):
         """Test that the ctrl is correctly taken into account"""
         res = qml.eigvals(qml.ctrl(qml.PauliX, 0))(wires=1)
         expected = np.linalg.eigvals(qml.matrix(qml.CNOT(wires=[0, 1])))
-        assert np.allclose(res, expected)
+        assert np.allclose(np.sort(res), np.sort(expected))

```

Comment:
Why do we obtain different ordering?

Reply:
`Controlled` operations' eigenvalues are ones followed by base operator eigenvalues.  The eigenvalues of `CNOT` are determined by `qml.math.linalg.eigvals` of the matrix.  The output of `np.linalg.eigvals` is not necessarily ordered.

So even though all the eigenvalues are the same, they were in different orders.